File: Programmers/src/숫자문자열과영단어.java
class Main {
    public int solution(String str) {
        String[] numTowords = new String[] {"zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"};
        
        for(int i=0;  i<numTowords.length; i++){
            str = str.replaceAll(numTowords[i], String.valueOf(i));
        }
        return Integer.parseInt(str);
    }   
}      

File: Programmers/src/Javascript/Lv1/숫자문자열과영단어.js
function solution(s) {
    var answer = s;
    let word = ['zero','one','two','three','four','five','six','seven','eight','nine'];
    
    for(let i=0; i<word.length; i++){
        let arr = answer.split(word[i]);
        answer = arr.join(i);
    }
    
    return Number(answer);
}